### PR TITLE
i18n(ko-KR): update `endpoints.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- update `endpoints.mdx`
- #9240

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by jsparkdev** [comment]
Once #10226, #10166 are merged, I expect to be able to merge this PR successfully.

**Review by yanthomasdev — APPROVED** [review]
LGTM! I'll merge the umbrella PR once I approve all of the individual PRs.

**Pull request closed without merging**
